src/nft/components/explore/Banner.tsx

```diff
@@ -0,0 +1,202 @@
+import clsx from 'clsx'
+import { Box } from 'nft/components/Box'
+import { Center, Column, Row } from 'nft/components/Flex'
+import { VerifiedIcon } from 'nft/components/icons'
+import { bodySmall, buttonMedium, header1, section } from 'nft/css/common.css'
+import { fetchTrendingCollections } from 'nft/queries'
+import { TimePeriod, TrendingCollection } from 'nft/types'
+import { formatEthPrice } from 'nft/utils/currency'
+import { putCommas } from 'nft/utils/putCommas'
+import { formatChange, toSignificant } from 'nft/utils/toSignificant'
+import { Dispatch, ReactNode, SetStateAction, useEffect, useReducer, useState } from 'react'
+import { useQuery } from 'react-query'
+import { useNavigate } from 'react-router-dom'
+
+import * as styles from './Explore.css'
+
+const Banner = () => {
+  const navigate = useNavigate()
+  /* Sets initially displayed collection to random number between 1 and 5  */
+  const [current, setCurrent] = useState(Math.floor(Math.random() * 5 + 1))
+  const [hovered, toggleHover] = useReducer((state) => !state, false)
```

Comment:
could you explain why hovered is being controlled vs using CSS alone?

Reply:
the hover state (on the entire banner) is used to pause the carousel rotation, and is also passed down to trigger an active state on the ‘Explore Collection’ button (even if the button itself is not hovered, per product spec). I think trying to both of those with CSS states would end up with more convoluted code